drivers/backup/helper/backup_helper.go

```diff
@@ -0,0 +1,90 @@
+package helper
+
+import (
+	"fmt"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	"github.com/portworx/torpedo/pkg/testrailuttils"
+	"github.com/portworx/torpedo/tests"
+	"gopkg.in/natefinch/lumberjack.v2"
+	"strconv"
+	"strings"
+)
+
+var (
+	pxBackupTorpedoTestInfoMap = make(map[int]*PxBackupTorpedoTestInfo, 0)
+)
+
+// PxBackupTorpedoTestInfo holds information of a particular test
+type PxBackupTorpedoTestInfo struct {
+	testName          string
+	testDescription   string
+	testAuthor        string
+	testRailID        int
+	testRunIdForSuite int
+	testTags          map[string]string
+	testLogger        *lumberjack.Logger
+}
+
+// StartPxBackupTorpedoTest creates a logger, configures the Aetos Dashboard for the specified test, and initializes controllers
+func StartPxBackupTorpedoTest(testRailId int, testName string, testDescription string, testAuthor string, apps []string, tags ...map[string]string) error {
```

Comment:
For some tests like `CreateMultipleUsersAndGroups` there are no `apps` deployed. How do we handle that?

Reply:
Pass empty array or nil.

Reply:
This may fail the API call to aetos if we have nil value

Reply:
Can you test this?

Reply:
https://go.dev/play/p/0q1VnWrKgqa